remotespark/livyclientlib/sparkcontroller.py

```diff
@@ -24,27 +27,27 @@ def __init__(self, serialize_path=None):
         else:
             self.client_manager = ClientManager()
 
-    def run_cell(self, client_name, context, cell):
-        context = context.lower()
-
-        # Select client
-        if client_name is None:
-            client_to_use = self.client_manager.get_any_client()
+    def run_cell(self, cell, client_name = None):
+        client_to_use = self.get_client_by_name(client_name)
+        (success, out) = client_to_use.execute(cell)
```

Comment:
I thought `(success, out)` should be returned to magics and the if/else below should be executed in the magics class?